en/docs/install-and-setup/setup/security/logins-and-passwords/maintaining-logins-and-passwords.md

```diff
@@ -66,9 +66,32 @@ Use the `<API-M_HOME>/bin/chpasswd.sh` script.
 
     If you are in the super tenant mode the username should be as in the format of `<email>@carbon.super`.
     ``` toml
+    [tenant_mgt]
```

Comment:
have these been added to the config catalog?

Reply:
[apim.throttling] properties are already added to the config catalog and the [tenant_mgt] will be added with this fix.